src/main/kotlin/com/personio/synthetics/config/BrowserTestConfig.kt

```diff
@@ -1,5 +1,11 @@
 package com.personio.synthetics.config
 
+/**
+ * Code here is mostly a copy of what is available in SyntheticTestBuilder,
+ * and will be deleted when 3.x.x is available.
```

Comment:
Perhaps, it's better to rephrase to something like: 

This code will be deprecated in the future releases as a duplicated one.
All the functions are already available in SyntheticTestBuilder.

Reply:
done